### ISSUE TITLE
zig master breakage

### ISSUE THREAD

**Issue by nektro** [zigtools/zls#452]
```
[meg@nixos:~/other/dev/zls]$ zig build
/home/meg/.local/share/zig/lib/std/hash_map.zig:346:13: error: Problems found with hash context type CompletionContext:
  CompletionContext.eql must be fn (self, CompletionItem, CompletionItem, b_index: usize) bool
    but is actually fn(analysis.CompletionContext, types.CompletionItem, types.CompletionItem) bool
            @compileError("Problems found with hash context type " ++ @typeName(Context) ++ ":" ++ errors);
            ^
/home/meg/.local/share/zig/lib/std/array_hash_map.zig:466:40: note: called from here
    comptime std.hash_map.verifyContext(Context, K, K, u32, true);
                                       ^
./src/analysis.zig:2304:52: note: called from here
pub const CompletionSet = std.ArrayHashMapUnmanaged(
                                                   ^
./src/analysis.zig:2304:52: note: called from here
pub const CompletionSet = std.ArrayHashMapUnmanaged(
                                                   ^
```

https://github.com/ziglang/zig/commit/3e99495ed8d2a384501338edb4885e709d51bf74

**Comment by nektro** [comment]
`0.10.0-dev.516+3e99495ed`

**Comment by nektro** [comment]
fixed by #453
